docs/platforms/android/Kotlin-android-studio-formatting.md

```diff
@@ -0,0 +1,27 @@
+# Setting up Kotlin formatting in Android Studio
+
+Kotlin code in this repository is formatted and linted with [`ktlint`](https://github.com/pinterest/ktlint).
+If you
+1. have submitted Kotlin code only to learn this when the analyzer check fails
+2. use Android Studio
+
+
+then there is good news! Android Studio can be configured to use `ktlint` to automatically apply formatting and highlight issues. To do this:
+
+
+1. Install the `ktlint` extension for Android Studio
+
+   a. On Mac, this is `Android Studio > Settings > Plugins > ` Search for `ktlint`.
+
+2. Set the ruleset to be the same as the version used in [`.ci.yaml`](../../../.ci.yaml) (as of writing this is 1.1.1), and the baseline to be `dev/bots/test/analyze-test-input/ktlint-baseline.xml`.
```

Comment:
Since .ci.yaml is at the root I thin you can use 
```suggestion
2. Set the ruleset to be the same as the version used in [`.ci.yaml`](.ci.yaml) (as of writing this is 1.1.1), and the baseline to be `dev/bots/test/analyze-test-input/ktlint-baseline.xml`.
```

Reply:
I think the current format is correct if you try clicking on it
https://github.com/flutter/flutter/blob/83d2e9c043a800b64c59c7c2c651d8825ef2549c/docs/platforms/android/Kotlin-android-studio-formatting.md